### PR TITLE
added the setting of an environment var and made directory writable

### PR DESCRIPTION
**Pull request by skanjila**
# Code Pull Requests

Please provide the following:

- set an environment variable identifying location of WANDB_DIR
- made the above directory writeable
- all local tests passing

### CODE REVIEW

File: tests/integration_tests/test_contrib_wandb.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 
 @pytest.mark.distributed
 def test_contrib_experiment(csv_filename):
+    os.environ["WANDB_DIR"] = os.path.join(os.getcwd(), 'results')
```
Comment (tgaddair) [inline tests/integration_tests/test_contrib_wandb.py:18]: Should use a tmpdir for this:

```
def test_contrib_experiment(csv_filename, tmpdir):
    wandb_dir = os.path.join(os.getcwd(), 'results')
    os.makedirs(wandb_dir)
    subprocess.call(['chmod', '-R', '+w', wandb_dir])
    os.environ["WANDB_DIR"] = wandb_dir
```

Comment (skanjila) [reply tests/integration_tests/test_contrib_wandb.py]: Updated to use tmp

Comment (tgaddair) [reply tests/integration_tests/test_contrib_wandb.py]: Hey @skanjila, the updated code has the same issue. Please see the updated reference code in my comment above (there was a typo before). The important thing to note is use of the `tmpdir` fixture.

Comment (skanjila) [reply tests/integration_tests/test_contrib_wandb.py]: Whoops sorry @tgaddair I changed the code to use tmpdir, let me know if anything else is missing

**Pull request closed without merging**